docs/sections/user_guide/yaml/components/ww3.rst

```diff
@@ -0,0 +1,24 @@
+.. _ww3_yaml:
+
+ww3
+===
+
+Structured YAML when including WaveWatchIII as part of a compiled coupled executable is validated by JSON Schema and requires the ``ww3:`` block, described below.
+
+Here is a prototype UW YAML ``ww3:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/ww3.yaml
+
+UW YAML for the ``ww3:`` Block
+------------------------------
+
+namelists:
+^^^^^^^^^^
+
+The path to the namelist(s) used by the WaveWatchIII configuration. Namelists will be symlinked to the run directory.
```

Comment:
Did we decide to render templates into the run directory instead of linking files?

Reply:
Ah, I didn't change this after implementing it to look more like the `fv3` driver on suggestion, which does currently render the templates rather than linking